fabric-renderer-indigo/src/client/java/net/fabricmc/fabric/impl/client/indigo/Indigo.java

```diff
@@ -89,7 +89,7 @@ private static TriState asTriState(String property) {
 	}
 
 	static {
-		File configDir = new File(FabricLoader.getInstance().getConfigDirectory(), "fabric");
+		File configDir = FabricLoader.getInstance().getConfigDir().resolve("fabric").toFile();
```

Comment:
Could possibly move this all to use NIO path's, but it makes no real difference, so dont worry about it for now 👍